### ISSUE TITLE
radio elements show blue focus ring on safari when selected via mouse

### ISSUE THREAD

**Issue by adixon-adobe** [adobe/spectrum-web-components#1426]
### Expected Behaviour
Blue focus borders only show up with keyboard navigation

### Actual Behaviour
On safari, these are showing up in radio elements when you click on them

#### Steps to Reproduce
Click one of the radio buttons here: https://opensource.adobe.com/spectrum-web-components/components/radio

#### Platform and Version
Tested on Safari 14.0.3 (15610.4.3.1.7, 15610)

**Comment by Westbrook** [comment]
@adixon-adobe I've got some testing I want to add, and I need to document the expanse of things this touches/fixes, but can you confirm that https://westbrook-focusable--spectrum-web-components.netlify.app/components/radio is performing as you expect?

**Comment by adixon-adobe** [comment]
What I'm seeing is that it's doing the right thing for clicks, but I'm unable to get keyboard focus at all on the radio documentation page (it looks like tab focus doesn't work before your fix, so likely a separate issue).
